force-app/main/default/objects/ServiceDelivery__c/fieldSets/Attendance_Service_Deliveries.fieldSet-meta.xml

```diff
@@ -51,6 +61,11 @@
         <isFieldManaged>false</isFieldManaged>
         <isRequired>false</isRequired>
     </availableFields>
+    <availableFields>
+        <field>UnitOfMeasurement__c</field>
+        <isFieldManaged>false</isFieldManaged>
+        <isRequired>false</isRequired>
+    </availableFields>
```

Comment:
This is a formula field. As long as r-e-f handles that as read-only, i guess it's ok. It's not useful since we handle it in the label of the quantity field like BSDT.